How we work animation

AvantikaSarkar opened the pull request:
Completed the following

1. Code cleanup/ refactoring
2. Mobile-view navbar issue
3. Added animation for remote-engg section mobile view
4. Added card layout for Calendly (contact-us section)

AvantikaSarkar committed:
changed relative paths to URL paths

AvantikaSarkar committed:
wip tjs_website/issue-fixes

- code cleanup/refactoring
- mobile view animation changes - remote engg section
- mobile view  navbar issue

priyankcommits commented on web/index.html:
```diff
@@ -1,36 +1,30 @@
-<!-- <!DOCTYPE html> -->
+ <!-- <!DOCTYPE html> -->
```
remove DS store checking in from git

priyankcommits commented on web/index.html:
```diff
@@ -58,110 +52,144 @@
             </div>
             <div class="col-sm-12 d-none d-sm-block d-md-none">
-                <img src="./assets/undraw_Freelancer_re_irh4.svg" alt="" height="300" width="500">
+                <img class="img-fluid" src="/assets/intro-section.svg" alt="" height="300" width="500">
```
You might not need inline height and width if you are usign bootstrap responsive classes

priyankcommits commented on web/index.html:
```diff
@@ -360,20 +374,19 @@ <h3 class="display-4"> Contact us...</h3>
-    <!-- <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" integrity="sha384-A7FZj7v+d/sdmMqp/nOQwliLvUsJfDHW+k9Omg/a/EheAdgtzNs3hpfag6Ed950n" crossorigin="anonymous"></script> -->
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js" integrity="sha384-vBWWzlZJ8ea9aCX4pEW3rVHjgjt7zpkNpZk+02D9phzyeVkE+jo0ieGizqPLForn" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js"
```
order of JS -> external -> local -> main

priyankcommits commented on web/main.js:
```diff
@@ -5,40 +5,39 @@ $(function () { // wait for document ready
-        .to("#slideContainer", 0.5, {z: -150})		// move back in 3D space
-        .to("#slideContainer", 1,   {x: "-14.286%"})	// move in to first panel
-        .to("#slideContainer", 0.5, {z: 0})				// move back to origin in 3D space
+        .to("#slideContainer", 0.5, { z: -150 })		// move back in 3D space
```
manage the spacing in JS files

priyankcommits commented on web/techStack.js:
```diff
@@ -0,0 +1,51 @@
+//Tech-stack GSAP animation
```
make everything ES5 or everything ES 6 syntax

AvantikaSarkar committed:
wip TJS website/Integrations

- changed spinner animation
- added progress line animation(wip)
- added Google analytics
- added meta tags for SEO
- integrated live chat
- added footer
- integrated privacy & cookies policy in footer

AvantikaSarkar committed:
Updated
